pages/linux/kwallet-query.md

```diff
@@ -0,0 +1,16 @@
+# kwallet-query
+
+> Read and write to a KDE Wallet.
+> More information: <https://manned.org/kwallet-query>.
+
+- List all entries in the Passwords folder of `kdewallet`:
```

Comment:
```suggestion
- List all entries in the passwords folder of `kdewallet`:
```

Reply:
No the folder is literally named "Passwords"

Reply:
```suggestion
- List all entries in the `Passwords` folder of `kdewallet`:
```

Okay, then maybe like this?